homeassistant/components/zimi/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "zimi",
+  "name": "zimi",
+  "codeowners": ["@markhannon"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/zimi",
+  "iot_class": "local_push",
+  "quality_scale": "bronze",
+  "requirements": ["StrEnum==0.4.15", "zcc-helper==3.3rc4"]
```

```suggestion
  "requirements": ["zcc-helper==3.3rc4"]
```
just use `from enum import StrEnum`